### PR TITLE
Qr/feature/setting up folder

### PR DESCRIPTION
**Pull request by qrulleau**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const defaultTheme = require('tailwindcss/defaultTheme');
+const colors = require('tailwindcss/colors');
+
+module.exports = {
+  purge: [
+    './index.html',
+    './src/**/*.{vue,js,ts,jsx,tsx}',
+    './node_modules/litepie-datepicker/**/*.js',
```
Comment (tbordinat) [inline tailwind.config.js]: You don't have the litepie-datepicker in your project (yet)

Comment (qrulleau) [reply tailwind.config.js]: my bad i will remove it :sweat_smile:

File: tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      gray: colors.blueGray,
+      red: colors.red,
+      yellow: colors.amber,
+      green: colors.emerald,
+      blue: colors.cyan,
+      indigo: colors.blue,
+      purple: colors.violet,
+      pink: colors.pink,
+      'litepie-primary': colors.lightBlue, // color system for light mode
+      'litepie-secondary': colors.coolGray, // color system for dark mode
```
Comment (tbordinat) [inline tailwind.config.js]: Same comment here

Comment (qrulleau) [reply tailwind.config.js]: bruh my bad :sweat:

**Pull request merged**